.github/workflows/trivy.yaml

```diff
@@ -1,6 +1,5 @@
 name: Security scan
 on:
-  pull_request:
```

Comment:
The idea here was to run `trivy.yaml` workflow on pull requests. It builds an image (don't push it) and checks it for vulnerabilities. 
On the other hand, `image.yaml` runs on push to main/new tag and builds a proper image and reports the security report.